application/models/SurveyDynamic.php

```diff
@@ -780,7 +780,7 @@ protected function filterColumns(CDbCriteria $criteria)
      * @param boolean $getComment
      * @return array | boolean
      */
-    public function getQuestionArray($oQuestion, $oResponses, $bHonorConditions, $subquestion = false, $getComment = false)
+    public function getQuestionArray($oQuestion, $oResponses, $bHonorConditions, $subquestion = false, $getComment = false, $sLanguage = null)
```

Comment:
When adding a new argument, don't forget to update the docblock too! :partying_face: